lib/evss/power_of_attorney_verifier.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module EVSS
+  class PowerOfAttorneyVerifier
+    def initialize(user)
+      @user = user
+      @use_mock = Settings.evss.power_of_attorney_verifier || false
```

Comment:
I *think* in most cases the settings specify whether to mock it or not (e.g. like `mock_vso_search`)

Also don't forget to make a devops pr for mocking in dev